todo_app/tests/test_endtoend.py

```diff
@@ -35,7 +35,7 @@ def app_with_temp_board():
     os.environ['LOGIN_DISABLED'] = 'True'
     db = get_board()
     # construct the new application
-    application = todo_app.app.create_app()
+    application = todo_app.app.create_app(db)
```

Comment:
Not directly relevant to these changes, but here you're passing through a DB object but I believe it should be identical to the one that the app would set up anyway, because you've overridden the `TRELLO_BOARD` environment variable. Could we simplify the test setup here by removing `get_board()` and just having that line:
```python
os.environ['TRELLO_BOARD'] = "test_board" + uuid.uuid1().
```

Reply:
Hi Jack, I have removed the passing of db, you are right that that is not necessary. 
I  still need to keep the get_board as i need to delete the board , therefore I need access to the database in the test app.